docs/serving/samples/cloudevents/cloudevents-spring/README.md

```diff
@@ -0,0 +1,139 @@
+# Spring + CloudEvents + Knative
+
+A simple web app written in Java using Spring Cloud Function that can receive CloudEvents. It
+supports running in two modes:
+
+1. The default mode has the app reply to your input events with the output
+   event, which is simplest for demonstrating things working in isolation, but
+   is also the model for working for the Knative Eventing `Broker` concept. The
+   input event is modified assigning a new source and type attribute.
+
+2. `K_SINK` mode has the app send events to the destination encoded in
+   `$K_SINK`, which is useful to demonstrate how folks can synthesize events to
+   send to a Service or Broker when not initiated by a Broker invocation (e.g.
+   implementing an event source). The input event is modified assigning a new
```

Comment:
In this mode, there's no input event? left over?

Reply:
I copied the README from another sample. So you're asking the wrong person, but there is an input event, in the sense that the application accepts input. It seems to work teh same as the other sample when I run it locally, so I might be wrong, but then we both are.

Reply:
I was confused, I thought this was acting as a source, in that there's no incoming event. You're right, sorry for the confusion :)